src/components/NFTBook/ItemCard.vue

```diff
@@ -161,11 +161,11 @@
     <div class="flex justify-between px-[8px] sm:px-[24px] mt-[20px]">
       <NFTBookTypeTags :content-types="contentTypes" />
       <template v-if="!isDetailsPreset">
-        <div v-if="nftIsCollectable">
+        <div v-if="nftBookFeaturedPrice">
           <Label
             preset="p5"
             class="text-like-green-dark"
-            :text="formattedNFTPriceInLIKE"
+            :text="nftBookFeaturedPrice"
```

Comment:
I think a better way is to check if book store price exists, if not show LIKE price if exists.

Reply:
The current `nftBookFeaturedPrice` has edition.price || listing.price.